client/src/Components/App/App.js

```diff
@@ -54,12 +57,16 @@ const App = () => {
 
   const updateLogs = () => {
     fetchCall()
-    .then(data => setUserLogs(data));
+    .then(data => setUserLogs(data))
+    .catch(err => {
+      setError(err);
+    });
```

Comment:
Love the simplicity in this! Kudos to @espressoGoddess, as well, for the insight into this!

Reply:
yeeehaw! so clean and simple